DataCapturing.podspec

```diff
@@ -39,6 +39,7 @@ This framework can be included by your App if you are going to capture sensor da
   # s.public_header_files = 'Pod/Classes/**/*.h'
   # s.frameworks = 'UIKit', 'MapKit'
   s.dependency 'ReachabilitySwift', '~> 4.1.0'
+  s.dependency 'Alamofire', '~> 4.6'
```

Comment:
Maybe add a comment what this dependency is used for (e.g. network communication) so that someone who reads the code for the first time does not need to google each dependency.